docs/zh/integration/offline_data_sources/s3.md

```diff
@@ -0,0 +1,27 @@
+# AWS S3
```

Comment:
Amazon S3
(I checked the official name should be Amazon S3)

Reply:
ok